kratos/python_scripts/python_solver.py

```diff
@@ -31,8 +31,23 @@ def __init__(self, model, settings):
         self.model = model
         self.settings = settings
 
+        # TODO remove the check once the derived solvers implement this
+        if hasattr(self, '_validate_settings_in_baseclass'):
```

Comment:
this ensures the backwards compatibility

Reply:
What if `_validate_settings_in_baseclass==False` ?

Reply:
doesn't matter it it is true or false, just needed sth to check.
Since this is TEMPORARY I didn't want to find a complex solution for sth that will be removed

Reply:
Simple solution would be to define it with false in the base class and overwrite it with true in the derived, but fine for me like this as well.

Reply:
I don't know where/how to define it in the baseclass since this would have to be done before the constructor of the baseclass it being called
=> hence I think we would need a more tricky solutions, which as said I thought is not worth it in this case

Reply:
Ok, now i see, sorry for the noise